generate/adapter_v2.py

```diff
@@ -65,7 +65,7 @@ def main(
                 device=fabric.device, dtype=dtype, quantization_mode=quantize
         ):
             model = LLaMA.from_name(name)
-            add_adapter_v2_parameters_to_linear_layers(model)
+            add_adapter_v2_parameters_to_linear_layers(model, dtype)
```

Comment:
Thanks for the update on the PR! Eager to give this a try!
Btw here I noticed that you'd also have to modify the `finetune/adapter_v2.py` script with so that it includes the `dtype` in the function call